src/current/_includes/common/upgrade/overview.md

```diff
@@ -1,18 +1,52 @@
+    - In the lead-up to a new major version's GA, a series of beta and RC releases {% if page.path contains "cockroachcloud" %}may be made available to CockroachDB {{ site.data.products.advanced }} as Pre-Production Preview releases{% else %}are made available{% endif %} for testing and validation. Testing releases are intended for testing and experimentation only, and are not qualified for production environments or eligible for support or uptime SLA commitments.
+
+      {% if page.path contains "cockroachcloud" %}If a cluster is upgraded to a Pre-Production Preview release, it will be automatically upgraded to subsequent patch releases within the major version, including newer Pre-Production Preview testing releases, the GA release, and subsequent production patch releases.{% else %}{{site.data.alerts.callout_info}}A cluster cannot be upgraded from an alpha binary of a prior release or from a binary built from the `master` branch of the CockroachDB source code.{{site.data.alerts.end}}{% endif %}
 
 To learn more about CockroachDB major versions and patches, refer to the [Releases Overview]({% link releases/index.md %}#overview).
+
+{% if page.path contains "cockroachcloud" %}
+### Upgrade differences across Cloud plans
+
+CockroachDB Cloud plan | Major version upgrades | Innovation releases
+---------------------- | ---------------------- | ----------------------
+Basic | Automatic | Required
+Standard | Automatic (default) or customer-initiated | Optional
+Advanced | Customer-initiated | Optional
+
+{{site.data.alerts.callout_info}}
+For CockroachDB Basic clusters and CockroachDB Standard clusters that are set to upgrade automatically, major version upgrades are finalized immediately and cannot be rolled back.
+{{site.data.alerts.end}}
+
+For all Cloud plans:
+
+- All major versions that are Regular releases (as opposed to Innovation releases) are required upgrades during the period in which they are supported.
```

Comment:
This sentence isn't clear. Regular releases aren't necessarily required.

Reply:
Refined to clarify that they're required in order to upgrade to a later major version, and separately present the fact that this needs to happen by EOS to maintain support.